backend/data/src/main/kotlin/io/tolgee/dtos/contentDelivery/S3ContentStorageConfigDto.kt

```diff
@@ -6,7 +6,7 @@ import jakarta.validation.constraints.Size
 
 class S3ContentStorageConfigDto : S3Config {
   @field:NotBlank
-  @field:Size(max = 255)
+  @field:Size(min = 3, max = 63)
```

Comment:
minor: I would put this change into a separate commit with description why it was done.

Or maybe even a separate PR if appropriate.